src/Sfa.Das.Sas.Infrastructure/Provider/Mapping/RoatpMapper.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Sfa.Das.Sas.Indexer.ApplicationServices.Provider.Models;
+using Sfa.Das.Sas.Indexer.ApplicationServices.Provider.Services;
+using Sfa.Das.Sas.Indexer.Core.Provider.Models;
+
+namespace Sfa.Das.Sas.Indexer.Infrastructure.Provider.Mapping
+{
+    public class RoatpMapper: IRoatpMapper
+    {
+        public List<RoatpProviderResult> Map(List<RoatpResult> roatpResults)
+        {
+            return roatpResults.Select(MapSingleRecord).ToList();
+        }
+
+        private static RoatpProviderResult MapSingleRecord(RoatpResult roatpResult)
+        {
+
+            ProviderType providerType;
+            switch (roatpResult.ProviderType)
```

Comment:
should we be ignoring the case?

Reply:
In reality, this is coming from a database that we totally control, so the case will always be the same, but I think making it case insensitive is reasonable.  'main provider' should be treated like 'Main provider', I think, so I'll make that change.